lib/phoenix_html.ex

```diff
@@ -12,6 +12,8 @@ defmodule Phoenix.HTML do
 
     * `Phoenix.HTML.Link` - functions for generating links and urls;
 
+    * `Phoenix.HTML.Format` - functions for formatting text;
```

Comment:
Oops, please update the `__using__` callback so we also import `Format` by default.